CHANGELOG.md

```diff
@@ -3,6 +3,14 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.3.0] - 2025-10-20
```

Comment:
Might want the date to be "Unreleased" or something like that, but it doesn't matter much as long as it's updated before releasing.